### ISSUE TITLE
And is not a function

### ISSUE THREAD

**Issue by ZHChen8397** [cucumber/cucumber-js#1016]
I got a problem when I use 'And' in my test.
The error shows that ' And is not a function '

Here is my feature

```
Feature: dataTable
  Scenario Outline: <a> + <b> + <c> = <answer>
    Given I had number <a>
    When I add with <b>
      And I add another number <c>
    Then I got answer <answer>
```

and my dataTable stepDefinition
```

const  assert = require('assert')
const defineSupportCode = require('cucumber').defineSupportCode;

defineSupportCode(function({Given,When,Then,And}){
  let ans = 0
  Given('I had number {int}', function(input){
    ans = input
  })
  When('I add with {int}',function(input){
    ans += input
  })
  And('I add another number {int}',function(input){
    ans += input
  })
  Then('I got answer {int}', function(input){
    assert.equal(ans,input)
  })
})
```
The error is 

```
TypeError: And is not a function
    at /Users/lab1321_mac_air/Desktop/cucumber/test/e2e/StepDefinition/dataTable.js:22:3
    at defineSupportCode (/Users/lab1321_mac_air/Desktop/cucumber/node_modules/cucumber/lib/support_code_library_builder/index.js:43:9)
    at Object.<anonymous> (/Users/lab1321_mac_air/Desktop/cucumber/test/e2e/StepDefinition/dataTable.js:4:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    // at my file location
```

I use cucumber@3.2.1 and 2.3.1 and both doesn't work
Is the  And definition remove from these version? 
If so, I wonder why we don't use 'And' anymore?  Thanks!

**Comment by charlierudolph** [comment]
There was never the ability to use And to create a step definition. I don't see any reason to add one anyway since Given / When / Then are all aliases for the same function that you use to define a step definition ([reasoning](https://github.com/cucumber/cucumber-js/blob/master/docs/faq.md#why-do-my-definition-patterns-need-to-be-globally-unique-instead-of-unique-only-within-given-when-then))

**Comment by ZHChen8397** [comment]
I got three new question here.

first,
The answer in faq.md, it says that 'If you have similar Given and Then patterns, try adding the word “should” to Then patterns.'
I don't understand clearly. To my example, I have two similar definition in When pattern, how could I use should in my Then pattern?

Second,
If we don't use And pattern. Is it correct that I modify my test like this?
`Feature: dataTable
  Scenario Outline: <a> + <b> + <c> = <answer>
    Given I had number <a>
    When I add with <b>
    When I add another number <c>
    Then I got answer <answer>`

Finally,
I studied the gherkin document, the original gherkin language had And Pattern. Is it because that you encourage a ubiquitous, non-ambiguous domain language, so you don't use And pattern?
